custom_components/ui_lovelace_minimalist/translations/de.json

```diff
@@ -0,0 +1,43 @@
+{
+    "title": "UI Lovelace Minimalist",
+    "config": {
+        "step": {
+            "user": {
+                "title": "UI Lovelace Minimalist Einstellungen",
+                "description": "",
+                "data": {
+                    "language": "Sprache",
+                    "sidepanel_enabled": "Erstellung eines Eintrages in der Seitenleiste aktivieren.",
+                    "sidepanel_icon": "Icon Seitenleiste",
+                    "sidepanel_title": "Titel Seitenleiste",
+                    "theme": "Thema",
+                    "include_other_cards": "Individuelle Karten, die für die Nutzung benötigt werden, ebenfalls laden."
+                }
+            }
+        },
+        "abort": {
+            "single_instance_allowed": "Es ist nur eine Konfiguration von UI Lovelace Minimalist erlaubt.",
+            "not_setup": "UI Lovelace Minimalist ist nicht eingerichtet."
+        }
+    },
+    "options": {
+        "step": {
+            "user": {
+                "title": "UI Lovelace Minimalist Einstellungen",
+                "description": "",
+                "data": {
+                    "language": "Sprache",
+                    "sidepanel_enabled": "Erstellung eines Eintrages in der Seitenleiste aktivieren.",
+                    "sidepanel_icon": "Icon Seitenleiste",
+                    "sidepanel_title": "Titel Seitenleiste",
+                    "theme": "Thema",
```

Comment:
Here aswell - "Stil"

Reply:
The official translation right now is "Thema", check your profile settings in HA for example. Tried to keep it coherent.

Reply:
Oh ok. In that case its fine the way it is👍